_overviews/scala3-migration/scaladoc-settings-compatibility.md

```diff
@@ -0,0 +1,55 @@
+| -doc-source-url | -source-links | A URL pattern used to link to the source file, with some variables supported... | Scala3 implementation provides richer syntax. | <i class="fa fa-check fa-lg"></i> |
+| -doc-external-doc | -external-mappings | Comma-separated list of classpath_entry_path#doc_URL pairs describing external dependencies. | Scala3 implementation provides richer syntax. | <i class="fa fa-check fa-lg"></i> |
+| -jdk-api-doc-base | -external-mappings | URL used to link Java API references. | You can specify jdk via -external-mappings since they are generalized setting. | <i class="fa fa-check fa-lg"></i> |
+| -doc-generator | _ | The fully qualified name of a doclet class, which will be used to generate the documentation | We don't need this in Scala3 |  <i class="fa fa-check fa-lg"></i> |
+| -doc-root-content | -doc-root-content | The file from which the root package documentation should be imported. | |  <i class="fa fa-check fa-lg"></i> |
+| -implicits | _ |  | We don't need this in Scala3 | <i class="fa fa-check fa-lg"></i> |
+| -implicits-debug | _ |  | We don't need this in Scala3 | <i class="fa fa-check fa-lg"></i> |
+| -implicits-show-all | _ |  | We don't need this in Scala3 |  <i class="fa fa-check fa-lg"></i> |
+| -implicits-sound-shadowing | _ |  | We don't need this in Scala3 |  <i class="fa fa-check fa-lg"></i> |
+| -implicits-hide | _ |  | We don't need this in Scala3 |  <i class="fa fa-check fa-lg"></i> |
+| -diagrams | _ |  | We don't need this in Scala3? | <i class="fa fa-check fa-lg"></i> |
+| -diagrams-debug | _ |  | We don't need this in Scala3? | <i class="fa fa-check fa-lg"></i> |
+| -diagrams-dot-path | _ |  | We don't need this in Scala3? |  <i class="fa fa-check fa-lg"></i> |
+| -diagrams-max-classes | _ |  | We don't need this in Scala3? |  <i class="fa fa-check fa-lg"></i> |
+| -diagrams-max-implicits | _ |  | We don't need this in Scala3? | <i class="fa fa-check fa-lg"></i> |
+| -diagrams-dot-timeout | _ |  | We don't need this in Scala3? | <i class="fa fa-check fa-lg"></i> |
+| -diagrams-dot-restart | _ |  | We don't need this in Scala3? |  <i class="fa fa-check fa-lg"></i> |
+| -author | _ |  | Fixed by [#11965](https://github.com/lampepfl/dotty/issues/11965) | <i class="fa fa-check fa-lg"></i> | 
+| -raw-output | _ |  | We don't need this in Scala3 | <i class="fa fa-check fa-lg"></i> |
+| -no-prefixes | _ |  | We don't need this in Scala3 | <i class="fa fa-check fa-lg"></i> |
+| -skip-packages | -skip-packages |  |  | <i class="fa fa-check fa-lg"></i> |
+| -no-link-warnings | _ | | | <i class="fa fa-check fa-lg"></i> |
+| -expand-all-types | _ | | Setting should be removed from Scala2 scaladoc | <i class="fa fa-check fa-lg"></i> |
```

Comment:
So, it’s not implemented? Should we mark it as such in the last column?

Reply:
Since it is supposed to be removed from old scaladoc (by the way it has not been removed even though it should have been in [Scala 2.13](https://github.com/scala/scala/blob/8a2cf63ee5bad8c8c054f76464de0e10226516a0/src/scaladoc/scala/tools/nsc/doc/Settings.scala#L218)) I assumed it is "implemented" by not being implemented at all.

Reply:
I understand the reasoning, but I think it would be clearer to the reader migrating from Scala 2 to Scala 3 to simply say that this setting has been removed in Scala 3.